templates/governance/group.hbs

```diff
@@ -2,25 +2,25 @@
 
 <header class="mt3 mt2-ns mb4 mb5-ns tc tl-ns">
   <div class="w-100 mw-none ph3 mw8-m mw9-l center f3">
-    <h1>{{ data.info[0].name }}</h1>
+    <h1>{{ data.info.[0].name }}</h1>
```

Comment:
did the data structure change? i'm curious about this

Reply:
Rocket upgraded from Handlebars 0.28 to 1.1 and that brought a breaking change to how array indexes are parsed (see https://github.com/sunng87/handlebars-rust/issues/246).